js/qmwp-mood-tracker.js

```diff
@@ -43,17 +43,17 @@ var onMoodButtonClicked = function () {
         combinationOperation: "MEAN",
         unit: "/5"
     }]
-    if (!access_token) {
+    if (typeof accessToken == "undefined" || !accessToken) {
         window.location.href = "?connect=quantimodo";
     } else {
         jQuery.ajax({
             type: "POST",
             data: JSON.stringify(mooddata),
-            url: api_host + "/api/measurements/v2",
+            url: (typeof apiHost !== 'undefined') ? apiHost + "/api/measurements/v2" : "/api/measurements/v2",
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'apiHost' is not defined.